src/includes/customizer/class-boldgrid-framework-customizer-typography.php

```diff
@@ -172,6 +172,15 @@ public function generate_font_size_css( $css = '' ) {
 	public function override_kirki_styles() {
 		$kirki_styles    = apply_filters( 'kirki_bgtfw_dynamic_css', Kirki_Modules_CSS::loop_controls( 'bgtfw' ) );
 		$filtered_styles = preg_replace( '/font-family:(\w+(\s\w+)+);/', 'font-family:"${1}";', $kirki_styles );
+
+		/*
+		 * Kirki no longer creates the kirki-css/styles.css file including their dynamic css.
+		 * Therefore we must create it for them in order to use the styles in the editor.
+		 */
+		if ( ! is_dir( wp_upload_dir()['basedir'] . '/kirki-css' ) ) {
```

Comment:
Use WP_Filesystem for these functions (mkdir,file_put_contents). Or Boldgrid_Framework_Wp_Fs